### PR TITLE
[15.0][MIG] partner_identification

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Comment by patrickrwilson** [comment]
@Freni-OSI This looks good but can you squash all the admin commits. Below is the reference to this requirement for migrating to 15.0...

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

"Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot. IMPORTANT: Don't squash legit translation commits, authored by their translators, with the message "Translated using Weblate (...)"."

**Comment by Freni-OSI** [comment]
> @Freni-OSI This looks good but can you squash all the admin commits. Below is the reference to this requirement for migrating to 15.0...
> 
> https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0
> 
> "Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot. IMPORTANT: Don't squash legit translation commits, authored by their translators, with the message "Translated using Weblate (...)"."

Done. please review.

**Review by patrickrwilson — APPROVED** [review]
Tested and LGTM

**Review by andhit-r — APPROVED** [review]
:heavy_check_mark: functional

File: partner_identification/models/res_partner_id_category.py [hunk 1]
```diff
@@ -58,14 +57,11 @@ def validate_id_number(self, id_number):
...
             safe_eval(self.validation_code, eval_context, mode="exec", nocopy=True)
         except Exception as e:
-            raise UserError(
-                _(
-                    "Error when evaluating the id_category validation code:"
-                    ":\n %s \n(%s)"
-                )
-                % (self.name, e)
-            )
+            msg = (
```
Comment (rousseldenis) [inline partner_identification/models/res_partner_id_category.py:60]: @Freni-OSI @dreispt This is not good !

You should format after the translation!

**Comment by rousseldenis** [comment]
@Freni-OSI Do you plan to fix this ?

**Comment by ValentinVinagre** [comment]
@Freni-OSI ??

File: partner_identification/models/res_partner_id_category.py (hunk 1 above)
Comment (andreschenkels) [reply partner_identification/models/res_partner_id_category.py:60]: @Freni-OSI will you be fixing this? Can I help?

Comment (Freni-OSI) [reply partner_identification/models/res_partner_id_category.py:60]: Can you check and review it again? Thanks

File: partner_identification/models/res_partner.py [hunk 2]
```diff
@@ -123,14 +123,11 @@ def _inverse_identification(self, field_name, category_code):
...
-                        "This %s has multiple IDs of this type (%s), so a write "
-                        "via the %s field is not possible. In order to fix this, "
-                        "please use the IDs tab."
-                    )
-                    % (record._name, category_code, field_name)
+                msg = "This {} has multiple IDs of this type {}, so a write via the {} field is\
+not possible. In order to fix this, please use the IDs tab.".format(
+                    record._name, category_code, field_name
                 )
+                raise ValidationError(_(msg))
```
Comment (rousseldenis) [inline partner_identification/models/res_partner.py:130]: @Freni-OSI Hich, you cannot do that!

You should format the result of translation function:

`raise ValidationError(_(msg).format(...))`

**Comment by dreispt** [comment]
Superseded by #1436

**Pull request closed without merging**